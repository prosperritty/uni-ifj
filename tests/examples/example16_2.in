-1230.123123