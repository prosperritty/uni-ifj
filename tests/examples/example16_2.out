Enter a number to compute its sum of squares: Invalid input.
